API: libpod/containers/<id>/stats for unprivileged mode unexpectedly succeeds and has wrong CPU and network numbers

Regression on RHEL 8.5.

Downstream report: https://bugzilla.redhat.com/show_bug.cgi?id=1988252

rhel-8-5
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testLifecycleOperationsUser (__main__.TestApplication)
Created symlink /etc/systemd/system/sockets.target.wants/podman.socket -> /usr/lib/systemd/system/podman.socket.
mkdir: cannot create directory '/home/admin/.ssh': File exists
Created symlink /home/admin/.config/systemd/user/sockets.target.wants/podman.socket -> /usr/lib/systemd/user/podman.socket.
[0730/074131.695367:ERROR:bus.cc(393)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
[0730/074131.696697:WARNING:headless_browser_main_parts.cc(83)] Cannot create Pref Service with no user data dir.

DevTools listening on ws://127.0.0.1:9886/devtools/browser/65302fe8-070e-4f3e-855f-0a1272a7ac43
[0730/074131.706483:ERROR:gpu_init.cc(426)] Passthrough is not supported, GL is egl
CDP: {"source":"security","level":"error","text":"Refused to apply inline style because it violates the following Content Security Policy directive: \"default-src 'self' http://127.0.0.2:9191\". Either the 'unsafe-inline' keyword, a hash ('sha256-BMIkowAqVEI88ibUJ1NtYB8ByZOuS4MTi73eGoFo2+Q='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.\n","timestamp":1627630894701.253,"url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/shell/index.js","lineNumber":30,"stackTrace":{"callFrames":[{"functionName":"ke","scriptId":"25","url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/shell/index.js","lineNumber":30,"columnNumber":35691},{"functionName":"Ze","scriptId":"25","url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/shell/index.js","lineNumber":30,"columnNumber":44728},{"functionName":"append","scriptId":"25","url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/shell/index.js","lineNumber":30,"columnNumber":46409},{"functionName":"D","scriptId":"25","url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/shell/index.js","lineNumber":63,"columnNumber":24932},{"functionName":"","scriptId":"25","url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/shell/index.js","lineNumber":63,"columnNumber":34130},{"functionName":"","scriptId":"25","url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/shell/index.js","lineNumber":68,"columnNumber":162663},{"functionName":"","scriptId":"25","url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/shell/index.js","lineNumber":68,"columnNumber":162667}]}}
> error: Modal: Specify at least one of: title, aria-label, aria-labelledby.
CDP: {"source":"network","level":"error","text":"Failed to load resource: the server responded with a status of 404 (Not Found)","timestamp":1627630894916.04,"url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/podman/po.js","networkRequestId":"28946.66"}
CDP: {"source":"network","level":"error","text":"Failed to load resource: the server responded with a status of 404 (Not Found)","timestamp":1627630895146.537,"url":"http://127.0.0.2:9191/cockpit/$b9bf1b0524fd7b03fe566f4629e484c32205c711f9224ea6c61b5bfd104b53f5/podman/po.js","networkRequestId":"28946.93"}
> warning: grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error

time="2021-07-30T03:41:36-04:00" level=warning msg="The input device is not a TTY. The --tty and --interactive flags might not work properly"
Traceback (most recent call last):
  File "/usr/lib64/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/work/bots/make-checkout-workdir/test/check-application", line 801, in testLifecycleOperationsUser
    self._testLifecycleOperations(False)
  File "/work/bots/make-checkout-workdir/test/check-application", line 862, in _testLifecycleOperations
    self.assertEqual(cpu, "n/a")
  File "/usr/lib64/python3.9/unittest/case.py", line 831, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib64/python3.9/unittest/case.py", line 1211, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib64/python3.9/unittest/case.py", line 670, in fail
    raise self.failureException(msg)
AssertionError: '0.00%' != 'n/a'
- 0.00%
+ n/a

Wrote screenshot to TestApplication-testLifecycleOperationsUser-rhel-8-5-127.0.0.2-2301-FAIL.png
Wrote HTML dump to TestApplication-testLifecycleOperationsUser-rhel-8-5-127.0.0.2-2301-FAIL.html
Wrote JS log to TestApplication-testLifecycleOperationsUser-rhel-8-5-127.0.0.2-2301-FAIL.js.log
Journal extracted to TestApplication-testLifecycleOperationsUser-rhel-8-5-127.0.0.2-2301-FAIL.log.gz
Warning: Stopping podman.service, but it can still be activated by:
  podman.socket
> warning: transport closed: disconnected
Traceback (most recent call last):
  File "/work/bots/make-checkout-workdir/test/check-application", line 801, in testLifecycleOperationsUser
    self._testLifecycleOperations(False)
  File "/work/bots/make-checkout-workdir/test/check-application", line 862, in _testLifecycleOperations
    self.assertEqual(cpu, "n/a")
AssertionError: '0.00%' != 'n/a'
- 0.00%
+ n/a


# Result testLifecycleOperationsUser (__main__.TestApplication) failed
# 1 TEST FAILED [44s on 4-cockpit-9]
not ok 7 /work/bots/make-checkout-workdir/test/check-application TestApplication.testLifecycleOperationsUser [ND@1]
```
First occurrence: 2021-07-30T07:42:00.884780 | revision a89e1ee60d83418415975140b82114fe476f1809
Times recorded: 1
Latest occurrences:

- 2021-07-30T07:42:00.884780 | revision a89e1ee60d83418415975140b82114fe476f1809

rhel-8-6
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testLifecycleOperationsUser (__main__.TestApplication)
Created symlink /etc/systemd/system/sockets.target.wants/podman.socket -> /usr/lib/systemd/system/podman.socket.
Created symlink /home/admin/.config/systemd/user/sockets.target.wants/podman.socket -> /usr/lib/systemd/user/podman.socket.
/usr/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
[0922/151243.905716:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory

DevTools listening on ws://127.0.0.1:9459/devtools/browser/1fd52035-b27a-45cf-9439-aff98aab6977
[0922/151243.917246:ERROR:gpu_init.cc(441)] Passthrough is not supported, GL is swiftshader
CDP: {"source":"security","level":"error","text":"Refused to apply inline style because it violates the following Content Security Policy directive: \"default-src 'self' http://127.0.0.2:9191\". Either the 'unsafe-inline' keyword, a hash ('sha256-BMIkowAqVEI88ibUJ1NtYB8ByZOuS4MTi73eGoFo2+Q='), or a nonce ('nonce-...') is required to enable inline execution. Note that hashes do not apply to event handlers, style attributes and javascript: navigations unless the 'unsafe-hashes' keyword is present. Note also that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.\n","timestamp":1632323566875.647,"url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/shell/index.js","lineNumber":1,"stackTrace":{"callFrames":[{"functionName":"ke","scriptId":"24","url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/shell/index.js","lineNumber":1,"columnNumber":120315},{"functionName":"Ze","scriptId":"24","url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/shell/index.js","lineNumber":1,"columnNumber":129352},{"functionName":"append","scriptId":"24","url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/shell/index.js","lineNumber":1,"columnNumber":131033},{"functionName":"D","scriptId":"24","url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/shell/index.js","lineNumber":1,"columnNumber":347890},{"functionName":"","scriptId":"24","url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/shell/index.js","lineNumber":1,"columnNumber":357088},{"functionName":"","scriptId":"24","url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/shell/index.js","lineNumber":1,"columnNumber":630305},{"functionName":"","scriptId":"24","url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/shell/index.js","lineNumber":1,"columnNumber":630309}]}}
> error: Modal: Specify at least one of: title, aria-label, aria-labelledby.
CDP: {"source":"network","level":"error","text":"Failed to load resource: the server responded with a status of 404 (Not Found)","timestamp":1632323567111.5461,"url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/podman/po.js","networkRequestId":"30815.67"}
CDP: {"source":"network","level":"error","text":"Failed to load resource: the server responded with a status of 404 (Not Found)","timestamp":1632323567332.0059,"url":"http://127.0.0.2:9191/cockpit/$5e00ef3ca681fd123aaa7b1e3cdb301fde7fbf4e4ff263575de6f174d6396d2d/podman/po.js","networkRequestId":"30815.94"}
> warning: grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error

time="2021-09-22T11:12:49-04:00" level=warning msg="The input device is not a TTY. The --tty and --interactive flags might not work properly"
Traceback (most recent call last):
  File "/usr/lib64/python3.9/unittest/case.py", line 550, in _callTestMethod
    method()
  File "/work/bots/make-checkout-workdir/test/check-application", line 826, in testLifecycleOperationsUser
    self._testLifecycleOperations(False)
  File "/work/bots/make-checkout-workdir/test/check-application", line 888, in _testLifecycleOperations
    self.assertEqual(cpu, "n/a")
  File "/usr/lib64/python3.9/unittest/case.py", line 829, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/usr/lib64/python3.9/unittest/case.py", line 1209, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib64/python3.9/unittest/case.py", line 668, in fail
    raise self.failureException(msg)
AssertionError: '0.00%' != 'n/a'
- 0.00%
+ n/a

Wrote screenshot to TestApplication-testLifecycleOperationsUser-rhel-8-6-127.0.0.2-2301-FAIL.png
Wrote HTML dump to TestApplication-testLifecycleOperationsUser-rhel-8-6-127.0.0.2-2301-FAIL.html
Wrote JS log to TestApplication-testLifecycleOperationsUser-rhel-8-6-127.0.0.2-2301-FAIL.js.log
Journal extracted to TestApplication-testLifecycleOperationsUser-rhel-8-6-127.0.0.2-2301-FAIL.log.gz
Warning: Stopping podman.service, but it can still be activated by:
  podman.socket
> warning: transport closed: disconnected
Traceback (most recent call last):
  File "/work/bots/make-checkout-workdir/test/check-application", line 826, in testLifecycleOperationsUser
    self._testLifecycleOperations(False)
  File "/work/bots/make-checkout-workdir/test/check-application", line 888, in _testLifecycleOperations
    self.assertEqual(cpu, "n/a")
AssertionError: '0.00%' != 'n/a'
- 0.00%
+ n/a


# Result testLifecycleOperationsUser (__main__.TestApplication) failed
# 1 TEST FAILED [48s on 1-cockpit-7]
not ok 7 /work/bots/make-checkout-workdir/test/check-application TestApplication.testLifecycleOperationsUser [ND@1]
```
First occurrence: 2021-09-22T15:13:13.339064 | revision 04005e2cc7693f9d0d41853f45e38c73604e4f98
Times recorded: 1
Latest occurrences:

- 2021-09-22T15:13:13.339064 | revision 04005e2cc7693f9d0d41853f45e38c73604e4f98

--- a/naughty/rhel-8/2259-podman-invalid-user-stats
+++ b/naughty/rhel-8/2259-podman-invalid-user-stats
@@ -1,6 +1,0 @@
-  File "test/check-application", line *, in testLifecycleOperationsUser
-    self._testLifecycleOperations(False)
-  File "test/check-application", line *, in _testLifecycleOperations
-    self.assertEqual(cpu, "n/a")
-*
-AssertionError: '0.00%' != 'n/a'

--- a/naughty/ubuntu-stable/2259-podman-invalid-user-stats
+++ b/naughty/ubuntu-stable/2259-podman-invalid-user-stats
@@ -1,6 +1,0 @@
-  File "test/check-application", line *, in testLifecycleOperationsUser
-    self._testLifecycleOperations(False)
-  File "test/check-application", line *, in _testLifecycleOperations
-    self.assertEqual(cpu, "n/a")
-*
-AssertionError: '0.00%' != 'n/a'

